package.json

```diff
@@ -24,6 +24,7 @@
     "npm": "3.10.10"
   },
   "devDependencies": {
+    "add-asset-html-webpack-plugin": "^1.0.2",
```

Comment:
are you opposed to pinning?

I've personally found it helpful in that version updates have specific testable PRs.

I guess shrink wrapping can have the same effect. thoughts?